simplesamlphp_auth.install

```diff
@@ -70,6 +70,7 @@ function simplesamlphp_auth_update_1000() {
   $config->set('users', update_variable_get('simplesamlphp_auth_users', ''));
   $config->set('logoutgotourl', update_variable_get('simplesamlphp_auth_logoutgotourl', ''));
   $config->set('user_register_original', update_variable_get('simplesamlphp_auth_user_register_original', '1'));
+  $config->set('link_display_name', update_variable_get('simplesamlphp_auth_link_display_name', 'Federated Log In'));
```

Comment:
`$config->set('login_link_display_name', update_variable_get('simplesamlphp_auth_login_link_display_name', 'Federated Log In'));`